frontend/src/app/pages/category-list/category-list.component.ts

```diff
@@ -65,9 +95,14 @@ export class CategoryListComponent implements OnInit {
       switch (res.event) {
         case "confirm-option":
           this.categoryService.deleteCategory(id).subscribe(() => {
-              this.getAllCategories(this.currentPage, this.pageSize);
+              this.categoryService.getAllCategories(
+                this.currentPage,
+                this.currentSize,
+                this.currentSortField,
+                this.currentDirection
+              );
               this.snackBar.open("Category was successfully deleted!", 'OK', {
-                duration:5000
+                duration: 5000
```

Comment:
you will often use `this.snackBar`, so it is reasonable to create `snackbarService` and have there 
```
    constructor(private snackbar: MatSnackBar) {}

    error(message: string) {
        this.snackbar.open(message, 'OK', {
            duration: SNACKBAR_CLOSE_ERROR_MILLIS,
            panelClass: ['snack-error'],
        });
    }

    success(message: string) {
        this.snackbar.open(message, 'OK', {
            duration: SNACKBAR_CLOSE_SUCCESS_MILLIS,
            panelClass: ['snack-success'],
        });
    }
```